packages/bitcore-client/bin/wallet-tx

```diff
@@ -8,23 +8,39 @@ program
   .option('--name <name>', 'REQUIRED - Wallet Name')
   .option(
     '--recipients <recipients>',
-    'REQUIRED - Address of receiver, or json array \'[{"address": "abc1234", "amount": xxxxxx}]\''
+    'REQUIRED - JSON array \'[{"address": "abc1234", "amount": xxxxxx}]\''
   )
-  .option('--fee <fee>', 'REQUIRED - The transaction fee to pay')
   .option('--utxos <utxos>', 'REQUIRED - Unspent transactions that can be spent')
   .option('--change <change>', 'REQUIRED - Change addresses')
+  .option('--fee [fee]', 'optional - custom transaction fee to pay')
+  .option('--target [target]', 'optional - custom target block for confirmation')
   .option('--path [path]', 'optional - Custom wallet storage path')
   .parse(process.argv);
 
 const main = async () => {
   const { name, path } = program;
   let wallet;
   try {
-    const { recipients, utxos, change, fee } = program;
+    const { recipients, utxos, change, fee, target=2 } = program;
     wallet = await Wallet.loadWallet({ name, path });
+
+    const parsedUtxos = utxos.includes('[') ? JSON.parse(utxos) : [];
+    const parsedRecipients = recipients.includes('[') ? JSON.parse(recipients) : [];
+
+    if (!parsedRecipients.length || !parsedUtxos.length) {
+      throw new Error('invalid arguments');
+    }
+
+    const calculatedNetworkFeeKb = await wallet.getNetworkFee({ target });
+    const netWorkFeeSat = calculatedNetworkFeeKb.feerate * 1E8/1000;
```

Comment:
Seems weird to have 1E8/1000

This is to convert the returned data from BTC to sats?

Reply:
yes, that's how it's done over frontend for insight @SonicWizard mentioned this